testbench/grpc_server.py

```diff
@@ -805,6 +805,41 @@ def RestoreObject(self, request, context):
         )
         return blob.metadata
 
+    @retry_test(method="storage.objects.move")
+    def MoveObject(self, request, context):
+        preconditions = testbench.common.make_grpc_preconditions(request)
+        bucket = self.db.get_bucket(request.bucket, context).metadata
+        src_object = self.db.get_object(
+            request.bucket,
+            request.source_object,
+            preconditions=preconditions,
+            context=context,
+        )
+        dst_metadata = storage_pb2.Object()
+        dst_metadata.CopyFrom(src_object.metadata)
+        dst_metadata.bucket = request.bucket
+        dst_metadata.name = request.destination_object
+        dst_metadata.metageneration = 1
```

Comment:
nit: I think this is handled in Object.init()
https://github.com/googleapis/storage-testbench/blob/e48a569f8f69e7dce432f83cf4e8341130873c2d/gcs/object.py#L98-L99

Reply:
Removed. Thanks Cathy.